MariaKarpava/units-tracking-app
# History screen

Changes Made:
- Added custom design to History screen.
- Implemented MVVM architecture for the History screen.
- Added custom fonts, colors and alignments for various UI elements.
- Added view states to handle different UI scenarios.
- Fixed bugs related to history items ordering and to units counting (taking into account newly added number of drinks).
- Added number of drinks section to the addNewDrink screen.
- Deleted unused code for better codebase cleanliness.


Test Plan:
- Add new drink when history is empty at start.
- Switch to the History screen and check that drink with correct info appeared on the screen with units counted correctly, taking into account quantity of the drink. 
- Add another drink and make sure that it appeared at the top of the list.
- When tapping on a cell, check that you was pushed to a temporary simple screen with some info about the chosen drink.
